src/index.html

```diff
@@ -208,19 +236,19 @@ <h2>Skills</h2>
 
     <footer class="navigation">
       <nav class="navigation__item">
-        <a href="#" data-js="quizappbutton" class="navigation__icon active">
+        <a href="#" data-js="quizapp-button" class="navigation__icon active">
```

Comment:
Please replace the anchor element with a button. You are not leaving the index.html with that element.